apps/web/src/app/api/workspaces/current/route.ts

```diff
@@ -0,0 +1,19 @@
+import { WorkspaceDto } from '@latitude-data/core/browser'
+import { authHandler } from '$/middlewares/authHandler'
+import { errorHandler } from '$/middlewares/errorHandler'
+import { NextRequest, NextResponse } from 'next/server'
+
+export const GET = errorHandler(
```

Comment:
Why you need this? We already have `useSession` which gives the current workspace

Reply:
I was told to do this change, because the workspace from the session was not updating after executing the update hook (store?), as it was always cached within the session. Notice that the already existing feature of updating the workspace name, had the same bug. It, however, showed the updated value in the UI because it was an input element.

Reply:
So, the thing is to have the useSession hook synchronized with the workspace store. Even initialize the workspace store on the layout backend entry point. But again, I'm not sure so let's talk briefly on Monday : )